docs/workflows/alternate-content-source.rst

```diff
@@ -0,0 +1,54 @@
+Alternate Content Sources
+=========================
+
+Alternate Content Sources can help speed up populating of new repositories.
+If you have content stored locally or geographically near you, it allows
+you its' faster transfer. Alternate Content Sources will allow you to use
+the content you have around if it matches the remote one.
```

Comment:
Rephrase the second half of this paragraph eg:

"If you have content stored locally or geographically near you which matches the remote content, Alternate Content Sources will allow you to substitute this content, allowing for faster data transfer."

(It doesn't have to be exactly like this, but it flows a little better when laid out this way)